doc/release/1.14.0-notes.rst

```diff
@@ -307,7 +307,15 @@ In summary, the major changes are:
 * User-defined ``dtypes`` (subclasses of ``np.generic``) now need to
   implement ``__str__`` and ``__repr__``.
 
-Some of these changes are described in more detail below.
+Some of these changes are described in more detail below. If you need to retain
+the previous behavior for doctests or other reasons, you may want to do
+something like::
+
+    # FIXME: Set numpy array str/repr to legacy behaviour on numpy > 1.13
```

Comment:
This comment seems a little weird to me. Maybe it should say something like "until we run our doctests on 1.14 and above"

Reply:
Reworded. In combination with the previous paragraph I think it is unlikely that users will use this by accident.